src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -67,6 +69,21 @@ class WorkspaceMembersPage extends React.Component {
         this.hideConfirmModal = this.hideConfirmModal.bind(this);
     }
 
+    componentDidMount() {
+        const clientPolicyMembers = _.keys(this.props.policyMemberList);
+        Policy.openWorkspaceMembersPage(this.props.policy.id, clientPolicyMembers);
+    }
+
+    componentDidUpdate(prevProps) {
+        const isReconnecting = prevProps.network.isOffline && !this.props.network.isOffline;
+        if (!isReconnecting) {
+            return;
+        }
+
+        const clientPolicyMembers = _.keys(this.props.policyMemberList);
+        Policy.openWorkspaceMembersPage(this.props.policy.id, clientPolicyMembers);
```

Comment:
```suggestion
        Policy.openWorkspaceMembersPage(this.props.policy.id, _.keys(this.props.policyMemberList));
```

Don't think we need that extra const definition since the variable name doesn't really provide any extra information here.

Reply:
Don't agree with this one. I think having the variable name makes the logic easier to read since it's not obvious what the keys of `policyMemberList` are. Also makes the lines shorter

Reply:
Ah, I see you name it the same thing as what you pass in the [Web PR](https://github.com/Expensify/Web-Expensify/pull/34718/files#diff-21986e7ed109aab3526d6eec36a22a130fec74691222e17c3b19c7588de38603R220). 

If that's the case then would you be opposed to renaming that variable to `clientPolicyMemberEmails` both here and in the Web PR? Feel like that's more descriptive than `clientPolicyMemberList`.

Reply:
I digggg, feel free to make that change here and in Web and I'll approve!

Reply:
cc ^ @arosiclair

Reply:
Change is in